From 91e99441d8a84211a5f22fc05f26bad6b8f07837 Mon Sep 17 00:00:00 2001
From: Tamar Christina <tamar.christina@arm.com>
Date: Wed, 28 Oct 2020 19:13:27 +0000
Subject: [PATCH 2/2] AArch64: Skip test for pr97535 on ILP32 since it can't
 express the range.

I am excluding the test from ILP32 since the goal of the test is to test
truncations of large numbers above INT_MAX.

gcc/testsuite/ChangeLog:

	PR target/97535
	* gcc.target/aarch64/pr97535.c: Exclude ILP32.
---
 gcc/testsuite/gcc.target/aarch64/pr97535.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/testsuite/gcc.target/aarch64/pr97535.c b/gcc/testsuite/gcc.target/aarch64/pr97535.c
index 6f83b3f5714..7d4db485f1f 100644
--- a/gcc/testsuite/gcc.target/aarch64/pr97535.c
+++ b/gcc/testsuite/gcc.target/aarch64/pr97535.c
@@ -1,4 +1,4 @@
-/* { dg-do compile } */
+/* { dg-do compile { target { ! ilp32 } } } */
 
 #include <string.h>
 
-- 
2.28.0

